### ISSUE TITLE
No feedback when clicking "Add to Pokrex"

### ISSUE THREAD

**Issue by sloria** [hlcfan/pokrex-chrome-extension#2]
It wasn't clear what the "Add to Pokrex" button was doing because there was no feedback after clicking it. I assumed that it was broken. In reality, the ticket gets added to a queue that is visible by clicking the extension button.

This can be made more clear with some visual feedback indicating what happened. Perhaps a notification that says "Queued for <ROOM NAME>" or something like that

**Comment by hlcfan** [comment]
Yes, you're right. I should've added the "click feedback", just didn't have much time while working on it.

I'll spare time on this and let you know, thanks!
